ml-algorithms/build.gradle

```diff
@@ -52,8 +52,8 @@ dependencies {
         exclude group: "com.microsoft.onnxruntime", module: "onnxruntime"
     }
     def os = DefaultNativePlatform.currentOperatingSystem
-    //mac doesn't support GPU
-    if (os.macOsX) {
+    //arm/macos doesn't support GPU
+    if (os.macOsX || System.getProperty("os.arch") == "aarch64") {
```

Comment:
What about Mac on Intel (pre M1/2)?

Reply:
that's why I left in the `if (os.macOsX` part.